ticdc: add two troubleshoot QAs (#4393)
pingcap/docs: ticdc/troubleshoot-ticdc.md

2. When `--tz` is not available, TiCDC tries to read the time zone set by the `TZ` environment variable.
3. When the `TZ` environment variable is not available, TiCDC uses the default time zone of the machine.

## What is the default behavior of TiCDC if I create a replication task without specifying the configuration file in `--config`?

> If you use the `cdc cli changefeed create` command without specifying the `-config` parameter, TiCDC creates the replication task according to the following steps:

Reviewer: ```suggestion
If you use the `cdc cli changefeed create` command without specifying the `-config` parameter, TiCDC creates the replication task in the following default behaviors:
```

> * Replicate all tables except system tables
> * Disable the Old Value feature
> * Skip replicating tables that do not contain [valid indexes](/ticdc/ticdc-overview.md#restrictions)

Reviewer: ```suggestion
* Replicates all tables except system tables
* Disables the Old Value feature
* Skips replicating tables that do not contain [valid indexes](/ticdc/ticdc-overview.md#restrictions)
```

| Null | 6 | `{"t":6,"v":null}` | |

For more information, refer to [TiCDC Open Protocol column type code](/ticdc/ticdc-open-protocol.md#column-type-code).

## How can I tell if a Row Changed Event of TiCDC Open Protocol is an `INSERT` event or an `UPDATE` event?

> If the the Old Value feature is not enabled, you cannot tell whether a Row Changed Event of TiCDC Open Protocol is an `INSERT` event or an `UPDATE` event. If the feature is enabled, you can determine the event type by the fields it contains:

Reviewer: ```suggestion
If the Old Value feature is not enabled, you cannot tell whether a Row Changed Event of TiCDC Open Protocol is an `INSERT` event or an `UPDATE` event. If the feature is enabled, you can determine the event type by the fields it contains:
```
